app/src/components/market/common/transaction_details_card/index.tsx

```diff
@@ -2,8 +2,15 @@ import React from 'react'
 import styled from 'styled-components'
 
 const Wrapper = styled.div`
-  border: 1px solid ${props => props.theme.borders.borderColor};
+  border: 1px solid ${props => props.theme.borders.borderDisabled};
+  background-color: ${props => props.theme.colors.mainBodyBackground};
   padding: 21px;
+  position: relaive;
+  border-bottom: 0;
+  background-image: url('data:image/svg+xml;utf8, <svg viewBox="0 0 200 110" xmlns="http://www.w3.org/2000/svg"><path d="M -15 110 L100 10 L215 110" fill="none" stroke="%23ECEFF1" stroke-width="1" vector-effect="non-scaling-stroke"/></svg>');
```

Comment:
I still think we should avoid using the background image, as I could see it resulting in unexpected behaviour.

> I followed that codepen sample and did it. But it's difficult to match the corners

It shouldn't be too hard to match the corners, why not just set the width of each element of the pattern to simply be the width of the div divided by some number?

Reply:
Setting the width of each element is not a good way.
It can break other layouts.
And currently, corners are matched correctly.

Reply:
![Screen Shot 2020-10-13 at 8 05 29 PM](https://user-images.githubusercontent.com/34900782/95928465-69b6bc80-0d8f-11eb-9088-c43a7b421be1.png)

vector-effect="non-scaling-stroke"

This matches the corner correctly

Reply:
![Screen Shot 2020-10-13 at 8 12 49 PM](https://user-images.githubusercontent.com/34900782/95928907-94554500-0d90-11eb-949a-864a871903af.png)

Reply:
https://stackoverflow.com/questions/53650302/container-with-zigzag-border-at-bottom-only-apply-to-the-border
@KadenZipfel , @pimato  found out this and I followed that to match the zigzag and corners.

Reply:
![Screen Shot 2020-10-14 at 5 30 34 AM](https://user-images.githubusercontent.com/34900782/95970789-5aab2b00-0dde-11eb-940a-c4432f0a839a.png)

@KadenZipfel , you're right.

I just set width of background-size to 2.5%.
In that case, that svg will be repeated 40 times exactly and the corners will be matched exactly.

Reference: https://stackoverflow.com/questions/53650302/container-with-zigzag-border-at-bottom-only-apply-to-the-border

Now, the corners are matched exactly. :) 
![Screen Shot 2020-10-14 at 5 32 31 AM](https://user-images.githubusercontent.com/34900782/95971013-a1992080-0dde-11eb-89f5-fde4173997f5.png)
![Screen Shot 2020-10-14 at 5 32 23 AM](https://user-images.githubusercontent.com/34900782/95971005-9fcf5d00-0dde-11eb-8b28-cbebda6b9d96.png)